package.json

```diff
@@ -1,3 +1,7 @@
 {
-    "version": "0.1.0"
-}
\ No newline at end of file
+    "version": "0.1.0",
+    "scripts": {
+	"version": "npm install && git add package.json package-lock.json",
+	"postversion": "echo '[~] pushing to `git@github.com:iScsc/iscsc.fr.git` please type your passphrase/password if required:' && git push git@github.com:iScsc/iscsc.fr.git $(git branch --show-current) v$(npm pkg get version | sed 's/\"//g') || echo \"[-] push failed, you can push with 'git push git@github.com:iScsc/iscsc.fr.git $(git branch --show-current) v$(npm pkg get version | sed 's/\"//g')'\""
```

Comment:
Okay you're right this part is really messy :sweat_smile: I'll modify it!

Reply:
Ok so after some research it looks like I can't write multilines in JSON format see [this StackOverflow thread](https://stackoverflow.com/questions/2392766/are-multi-line-strings-allowed-in-json)
So too bad...
And I don't like the idea of letting another `script.sh` at root, not designed to be used by hand :confused:
So let me know what you think? (@atxr too)

Reply:
We will stick with JSON sorry even if it's bad :laughing: 
But adding a `script.sh` in a script folder at root level sounds good for me

Reply:
Actually as you can read on the forum I linked JSON isn't designed to be human readable so it isn't better or worse than `yaml` or `toml` just serves different goal!
If the solution of another script seems fine to you I'll write it! But place it in `./scripts/private` I think! Because it isn't designed to be run by hand!

Reply:
In the end I preferred to throw the postversion script to the bin and put the code in `bump.sh`. I think that makes much more sense :+1:

Reply:
in the end, it's not a big deal :thinking: 

if @atxr, you'd like a separate script, then @ctmbl please implement it :+1: 
otherwise, the `\n` are fine with me, the script isn't that big :thinking:

Reply:
it's been implemented in `8a95601..d681cf5` right? :eyes:

Reply:
Absolutely!

Reply:
perfect :ok_hand: